ci/testnet-deploy.sh

```diff
@@ -128,6 +120,14 @@ while getopts "h?p:Pn:c:t:gG:a:Dbd:rusx" opt; do
   esac
 done
 
+netName=${@:$OPTIND:1}
+cloudProvider=${@:$OPTIND+1:1}
+zone=${@:$OPTIND+2}
+
+[[ -n $netName ]] || usage "Network name not specified"
+[[ -n $cloudProvider ]] || usage "Cloud provider not specified"
+[[ -n $zone ]] || usage "Zone not specified"
```

Comment:
@danpaul000 , the "zone" can contain a list of zones now. The list needs to be split up into individual zone, and passed to cloudProvider.sh  script as -z zone1 -z zone2 ... -z zoneN.

Maybe it'll be best if we change this script to take zone not as a positional argument. We can pass each zone separately, with -z (similar to gce.sh). This will also leave other positional argument at their original place, reducing the need for change in testnet-manager.sh.